migrations/006-create-notification-table.js

```diff
@@ -0,0 +1,40 @@
+const tableName = 'Notifications';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable(tableName, {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4
+      },
+      title: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: false
+      },
+      description: {
```

Comment:
Could you make this `message` please? I feel it'd be more in line with the idea of a notification. Just a name, but makes things clearer.